packages/color-kit-block/src/ColorKitBlock.spec.ct.tsx

```diff
@@ -0,0 +1,17 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from '@cypress/react';
+import { withAppBridgeBlockStubs } from '@frontify/app-bridge';
+
+import { ColorKitBlock } from './ColorKitBlock';
+
+const ColorKitBlockSelector = '[data-test-id="color-kit-block"]';
+
+describe('Color Kit block', () => {
```

Comment:
Missing tests

Reply:
in progress. As I did some code splitting.